src/main/java/com/echobox/github/cycletime/data/PullRequestCSVDAO.java

```diff
@@ -72,10 +75,26 @@ public PullRequestCSVDAO(String filename, ZoneId persistWithTimezone, boolean ap
    * @throws IOException If the write fails
    */
   public synchronized void writeCSVHeader() throws IOException {
-    csvWriter.write("UTCMergedDateTime,MonthIndex,Repo-PRNum,Title,PRAuthor,CodingTimeSecs,"
-        + "PickupTimeSecs,ReviewTimeSecs,Review1,Review2,Review3,Review4,Review5,Review6\n");
+    writeCSVHeader(new ArrayList<>());
   }
   
+  /**
+   * Write the CSV header for PR analysis
+   * @param additionalHeaders Append the additional heads to the end of the standard ones
+   * @throws IOException If the write fails
+   */
+  public synchronized void writeCSVHeader(List<String> additionalHeaders) throws IOException {
+    
+    String headers = "UTCMergedDateTime,MonthIndex,Repo-PRNum,Title,PRAuthor,CodingTimeSecs,"
+        + "PickupTimeSecs,ReviewTimeSecs,Review1,Review2,Review3,Review4,Review5,Review6";
+    
+    if (additionalHeaders != null) {
```

Comment:
I had to add this to get this to work, otherwise it appended a `,` at the end and made an empty header that did not pass validation:
```suggestion
    if (additionalHeaders != null && !additionalHeaders.isEmpty()) {
```